site/_docs/history.md

```diff
@@ -46,19 +106,193 @@ other software versions as specified in gradle.properties.
 #### New features
 {: #new-features-1-31-0}
 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4865">CALCITE-4865</a>]
+ Allow table functions to be polymorphic
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5194">CALCITE-5194</a>]
```

Comment:
It seems that there are several items in this section (such as this one: CALCITE-5194), which are not in fact "new features" but rather bug fixes (they are categorized as bugs in Jira), so perhaps they should be moved into the "Bug-fixes, ..." section. Could you please review it?

Reply:
Thanks for pointing it out.